### ISSUE TITLE
ActiveStorage content_type should not be nil

### ISSUE THREAD

**Issue by dcdieci** [rails/rails#31708]
### Steps to reproduce
Create a new attached file for an arbitrary record like an user without providing the content-type which is currently allowed

`User.last.images.attach(io: File.open(Rails.root.join('public','image.png')) , filename: 'image.png' )
`

### Expected behavior
a.) it should raise an error because of the missing content type
or
b.) don't rely on content type in 
ActiveStorage::Blob
e.g. line 94
```
  def image?
    content_type.start_with?("image")
  end
```

### Actual behavior
content_type is nil and thus the ImageAnalyzer fails

` NoMethodError: undefined method `start_with?' for nil:NilClass
`
```
2018-01-14T20:09:10.203Z 38019 TID-ow9v73maw WARN: NoMethodError: undefined method `start_with?' for nil:NilClass
2018-01-14T20:09:10.203Z 38019 TID-ow9v73maw WARN: /Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:95:in `image?'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/lib/active_storage/analyzer/image_analyzer.rb:16:in `accept?'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:311:in `block in analyzer_class'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:311:in `each'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:311:in `detect'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:311:in `analyzer_class'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:307:in `analyzer'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:303:in `extract_metadata_via_analyzer'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/models/active_storage/blob.rb:252:in `analyze'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activestorage-5.2.0.beta2/app/jobs/active_storage/analyze_job.rb:6:in `perform'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activejob-5.2.0.beta2/lib/active_job/execution.rb:39:in `block in perform_now'
/Users/dcdieci/.rvm/gems/ruby-2.4.3/gems/activesupport-5.2.0.beta2/lib/active_support/callbacks.rb:109:in `block in run_callbacks'
```

### System configuration
**Rails version**:
5.2.0.beta2/
**Ruby version**:
ruby 2.4.3p205 (2017-12-14 revision 61247) [x86_64-darwin16]

**Comment by georgeclaghorn** [comment]
This will be fixed by #31693.

**Comment by georgeclaghorn** [comment]
Closing because #31693 was merged.
